mkosi/__init__.py

```diff
@@ -2549,17 +2549,17 @@ def run_verb(args: MkosiArgs, presets: Sequence[MkosiConfig]) -> None:
         if last.compress_output:
             die(f"Sorry, can't {opname} a compressed image.")
 
-    for config in presets:
+    for i in images:
```

Comment:
Here and everywhere else: `i` is very short and at least to me will always just be some integer. :) How about 
```suggestion
    for img in images:
```
`img` is also much easier to grep for then `i`.

Reply:
Switched back to `config` instead. `image.image` is a bit weird to read.